functional/tests/core-manage-discovery-jsonrpc.test.mjs

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+import { expect, jest, test, afterAll, beforeAll } from "@jest/globals";
+import * as utilities from "./utilities.mjs";
+
+jest.setTimeout(30040);
```

Comment:
use constant timeout. May be move '30040' to constants.js and can be imported and call the constant value.

Reply:
I updated the timeout to 20s and didn't include it any constant file as we are using it here only.